dictionaries/CallMap.php

```diff
@@ -346,7 +346,7 @@
 'ArgumentCountError::getLine' => ['int'],
 'ArgumentCountError::getMessage' => ['string'],
 'ArgumentCountError::getPrevious' => ['?Throwable'],
-'ArgumentCountError::getTrace' => ['list<array<string,mixed>>'],
+'ArgumentCountError::getTrace' => ['list<array{file:string,line:int,function:string,class?:string,type?:\'::\'|\'->\',args?:array<mixed>}>'],
```

Comment:
per https://github.com/phpstan/phpstan-src/pull/914#issuecomment-1007370554 `file` needs to be nullable

Reply:
and `class` can be narrowe to `class-string`

Reply:
`line` is optional as well apparently.